ag/ag.proto

```diff
@@ -445,4 +456,5 @@ service AutograderService {
     rpc GetOrganization(OrgRequest) returns (Organization) {}
     rpc GetRepositories(URLRequest) returns (Repositories) {}
     rpc IsEmptyRepo(RepositoryRequest) returns (Void) {}
+    rpc GetStudentForDiscord(DiscordRequest) returns (DiscordResponse) {}
```

Comment:
Suggest this:

```proto
    rpc GetUserByCourse(CourseUserRequest) returns (User) {}
```

Where `CourseUserRequest = DiscordRequest`, and `User` is the same as our current `User`. The remote ids are removed before the result is returned anyway. The Discord bot could maybe use the same avatar URL and admin field.